homeassistant/components/whirlpool/config_flow.py

```diff
@@ -108,6 +108,7 @@ async def async_step_reauth_confirm(
             step_id="reauth_confirm",
             data_schema=REAUTH_SCHEMA,
             errors=errors,
+            description_placeholders={"name": "Whirlpool"},
```

Comment:
If I look in `strings.json` there seems to be nothing that uses it?

Reply:
I guess I was looking at the strings.json for a different integration (honeywell) - should I leave the title as is or move to the common title and add the key?

Reply:
I think it's nice to move to the common title.